streams/src/test/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyKeyValueStoreTest.java

```diff
@@ -257,6 +257,19 @@ public long approximateNumEntries() {
         assertEquals(Long.MAX_VALUE, theStore.approximateNumEntries());
     }
 
+    @Test
+    public void shouldReturnLongMaxValueOnUnderflow() {
+        stubProviderTwo.addStore(storeName, new NoOpReadOnlyStore<Object, Object>() {
+            @Override
+            public long approximateNumEntries() {
+                return Long.MIN_VALUE;
```

Comment:
I would return `MAX_VALUE` here -- we should also add multiple stores that do return `MAX_VALUE` to test the inner part of the loop.

Reply:
Thanks will take care!